SwiftGen.playground/Pages/Fonts-Demo.xcplaygroundpage/Contents.swift

```diff
@@ -40,7 +40,7 @@ struct FontFamily {
 // Using the UIFont constructor…
 let helvetica = UIFont(font: FontFamily.Helvetica.Regular, size: 20.0)
 // Or using the enum value and its `font` method
-let helveticaNeue = FontFamily.HelveticaNeue.Regular.font(20.0)
+let helveticaNeue = FontFamily.HelveticaNeue.Regular.font(size: 20.0)
 
-let helveticaBoldBig = FontFamily.Helvetica.Bold.font(100.0)
+let helveticaBoldBig = FontFamily.Helvetica.Bold.font(size: 100.0)
```

Comment:
Don't we generate lowercase cases now, like `Helvetica.bold` and not `Helvetica.Bold`? I mean, if the Playground is in Swift 3 (which this `font(size: 100)` instead of `font(100)` seems to suggest, should we assume that the code in the Playground might reflect the code from the `-t swift3` template, or should we still use the code generated by the default template in the Swift 3 playground?